closure/goog/date/date.js

```diff
@@ -1664,6 +1664,18 @@ goog.date.DateTime.prototype.equals = function(other) {
 };
 
 
+/**
+ * Tests whether given time part of datetime is exactly equal to time part of this DateTime.
+ *
+ * @param {goog.date.DateTime} other The datetime to compare.
+ * @return {boolean} Whether the given time part of datetime is exactly equal to time part of this one.
+ * @override
+ */
+goog.date.DateTime.prototype.timeEquals = function(other) {
```

Comment:
My two cents:

- There's a bunch of isSame*blah* static methods. This fits in with those.
- Using "time" by itself is confusing here. Perhaps refer to it as time ignore date? e.g. isSameIgnoreDateMonthAndYear or something?